### PR TITLE
Helm chart for the PagerDuty plugin

### PR DESCRIPTION
**Pull request by kbence**
This change adds a basic Helm chart and documentation for the PagerDuty plugin

Closes #495

### CODE REVIEW

File: charts/access/pagerduty/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <td>string</td>
+    <td><code>""</code></td>
+    <td>yes</td>
+  </tr>
+  <tr>
+    <td><code>teleport.identitySecretName</code></td>
+    <td>Name of the Kubernetes secret that contains the credentials for the connection</td>
+    <td>string</td>
+    <td><code>""</code></td>
+  </tr>
```
Comment (wadells) [inline charts/access/pagerduty/README.md:30]: This row and all below are missing a `<td>`for the final "Required" column.

File: charts/access/pagerduty/values.schema.json [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+        "log"
+    ],
+    "properties": {
+        "image": {
+            "$id": "#/properties/image",
+            "type": "object",
+            "default": {},
+            "examples": [
+                {
+                    "repository": "146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-plugin-pagerduty",
```
Comment (wadells) [inline charts/access/pagerduty/values.schema.json:27]: Use the new quay.io link.

File: charts/access/pagerduty/values.schema.json [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+                "pullPolicy",
+                "tag"
+            ],
+            "properties": {
+                "repository": {
+                    "$id": "#/properties/image/properties/repository",
+                    "type": "string",
+                    "default": "146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-plugin-pagerduty",
+                    "examples": [
+                        "146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-plugin-pagerduty"
```
Comment (wadells) [inline charts/access/pagerduty/values.schema.json:43]: use the new quay.io link

File: charts/access/pagerduty/values.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# Default values for pagerduty.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# replicaCount: 1
+
+image:
+  repository: 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-plugin-pagerduty
```
Comment (wadells) [inline charts/access/pagerduty/values.yaml:8]: Use the new quay repo.

**Review by wadells — COMMENTED** [review]
Same commend as #494: This needs to be tied into the general Makefile as well as the CI you set up in https://github.com/gravitational/teleport-plugins/pull/459.

**Review by wadells — APPROVED** [review]
:ship:

**Review by Joerger — APPROVED** [review]

**Pull request merged**